src/BoxAnnotations.js

```diff
@@ -19,7 +19,7 @@ const ANNOTATORS = [
         CONSTRUCTOR: DocAnnotator,
         VIEWER: ['Document', 'Presentation'],
         TYPE: [TYPES.point, TYPES.highlight, TYPES.highlight_comment, TYPES.draw],
-        DEFAULT_TYPES: [TYPES.point, TYPES.highlight, TYPES.highlight_comment]
+        DEFAULT_TYPES: [TYPES.point, TYPES.highlight, TYPES.highlight_comment, TYPES.draw]
```

Comment:
Can this be a part of a separate commit?

Reply:
I don't plan on merging this patch.